docs/modules/PaperSize.rst

```diff
@@ -0,0 +1,7 @@
+The PaperSize Class
+------------------------
```

Comment:
```suggestion
The PaperSize Class
-------------------
```

I vaguely remember that there were warnings if the lines didn't match ... even if there are not, it looks nicer :-)